multiraft/multiraft.go

```diff
@@ -644,10 +644,7 @@ func (s *state) start() {
 	})
 }
 
-func (s *state) removePending(g *group, prop *proposal, err error) {
-	if prop == nil {
-		return
-	}
+func (prop *proposal) removeFromPending(g *group, err error) {
```

Comment:
Yeah, you're right, it probably makes more sense to make that the receiver. Done.